src/main/java/br/com/senior/hcm/payroll/pojos/Pagination.java

```diff
@@ -0,0 +1,30 @@
+package br.com.senior.hcm.payroll.pojos;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+/**
+ * Dados da paginação
+ */
+@Data
+@AllArgsConstructor
+@RequiredArgsConstructor
+public class Pagination {
+    /**
+     * "Página atual, inicia em 1"
+     */
+    @NonNull
```

Comment:
Está com este warning:
@NonNull is meaningless on a primitive.

Tem como remover isso?

No atributo "size" tbém ocorre

Reply:
Pra mim este problema não está ocorrendo, qual versão do lombok que você está utilizando?

Reply:
versão 1.18.10